_posts/2019-07-18-snowplow-r115-sigiriya-eer/index.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title-short: "Snowplow R115 with minor updates to EmrEtlRunner and Event Manifest Populator"
+title: "Snowplow R115 with minor updates to EmrEtlRunner and Event Manifest Populator"
+tags: [snowplow, emretlrunner, release]
+author: Ben Benoist
+category: Releases
+permalink: /blog/2019/07/18/snowplow-r115-sigiriya-emretlrunner/
+---
+
+We are pleased to release [Snowplow R115 Sigiriya][snowplow-release], named after
+[the ancient ancient rock fortress in Sri Lanka](https://en.wikipedia.org/wiki/Sigiriya#Archaeological_remains_and_features). This Snowplow release includes two updates to EmrEtlRunner and one to Event Manifest Populator (used for cold start deduplication).
+
+1. [EmrEtlRunner](#eer)
+2. [Event Manifest Populator](#emp)
+3. [Upgrading](#upgrading)
+4. [Roadmap](#roadmap)
+5. [Getting help](#help)
+
+<h2 id="eer">1. Updates to EmrEtlRunner</h2>
+
+<h3>1.1. Bug fix to the function displaying failures</h3>
+
+While improving the reliability of [EmrEtlRunner](https://github.com/snowplow/snowplow/tree/master/3-enrich/emr-etl-runner) in R114, a bug had been introduced where a step could fail without having the error message in the logs.
+
+This has now been fixed and we improved the code quality of `EmrEtlRunner` by adding more unit testing.
+
+<h3>1.2. Step failure on transient EMR cluster</h3>
+
+Still in an effort to improve the reliability of `EmrEtlRunner`, an update has been made so that `EmrEtlRunner` fails if an EMR step can't be successfully submitted to a transient EMR cluster.
+
+This was already the case for a standard EMR cluster.
+
+<h2 id="emp">2. Event Manifest Populator</h2>
+
+To solve the "cold start" problem for cross-batch deduplication in the RDB Shredder, we had developed the [Event Manifest Populator](https://github.com/snowplow/snowplow/tree/master/5-data-modeling/event-manifest-populator).
+
+It was developped to use events emitted by `spark-enrich` as input, and we now added the possibility to also read events emitted by `stream-enrich`.
```

Comment:
Is the Event Manifest Popular something that _was_ added and is now being updated? Or is it new in this release? It's not 100% clear.

Reply:
It was added and is now being updated. I will update to make it more clear.